### ISSUE TITLE
debug output on stdout

### ISSUE THREAD

**Issue by aucampia** [mvnx/mvnx#7]
Currently mvnx puts debug output on stdout instead of stderr. This is incorrect use of stdout:

 * https://en.wikipedia.org/wiki/Standard_streams#Standard_error_(stderr)
 * https://stackoverflow.com/questions/3385201/confused-about-stdin-stdout-and-stderr

While the tool is nice, this makes the tool rather useless for any situation where you care about the output, which is quite often.

```
20210210T190227 iwana@teekai.zoic.eu.org:~
$ mvnx com.github.ricksbrown:cowsay "Hello, World!" 1>/dev/null 
20210210T190232 iwana@teekai.zoic.eu.org:~
$ mvnx com.github.ricksbrown:cowsay "Hello, World!" 2>/dev/null 
Searcing for artifact in the local repository at
  /home/iwana/.m2/repository/com/github/ricksbrown/cowsay/1.1.0/cowsay-1.1.0.jar
Artifact found in the local repository at
  /home/iwana/.m2/repository/com/github/ricksbrown/cowsay/1.1.0/cowsay-1.1.0.jar
Executing JAR
  java -jar /home/iwana/.m2/repository/com/github/ricksbrown/cowsay/1.1.0/cowsay-1.1.0.jar Hello, World!
 _______________
< Hello, World! >
 ---------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||

```

**Comment by battila7** [comment]
Hi,

thanks for opening this issue! You're right, mvnx does not make correct use of stdio.  I will soon release a new version featuring the correct separating of debug/tracing output from the JAR execution output.

**Comment by battila7** [comment]
@aucampia 
Released version [v4.1.0](https://github.com/mvnx/mvnx/releases/tag/v4.1.0) on both GitHub and npm. In the new version, stdout and stderr are treated correctly, as you can verify, for example, by executing

~~~~
npx mvnx@4.1.0 com.github.ricksbrown:cowsay "Hello, World!" > output
~~~~

which outputs

~~~~
npx: installed 1 in 1.412s
Searcing for artifact in the local repository at
  C:\Users\Attila\.m2\repository\com\github\ricksbrown\cowsay\1.1.0\cowsay-1.1.0.jar
Artifact found in the local repository at
  C:\Users\Attila\.m2\repository\com\github\ricksbrown\cowsay\1.1.0\cowsay-1.1.0.jar
Executing JAR
  java -jar C:\Users\Attila\.m2\repository\com\github\ricksbrown\cowsay\1.1.0\cowsay-1.1.0.jar Hello, World!
~~~~

Please let me know, if things work out for you now. Also, if everything's fine, please close the issue. Thanks!

**Comment by battila7** [comment]
Closing because of inactivity.
